openedx/features/course_experience/url_helpers.py

```diff
@@ -161,14 +163,19 @@ def make_learning_mfe_courseware_url(
     """
     mfe_link = f'{settings.LEARNING_MICROFRONTEND_URL}/course/{course_key}'
 
+    if preview:
+        mfe_link = f'{settings.LEARNING_MICROFRONTEND_URL}/preview/course/{course_key}'
+
     if sequence_key:
         mfe_link += f'/{sequence_key}'
 
         if unit_key:
             mfe_link += f'/{unit_key}'
 
     if params:
-        mfe_link += f'?{params.urlencode()}'
+        get_params = params.copy()
+        del get_params['preview']
+        mfe_link += f'?{get_params.urlencode()}'
```

Comment:
Same nit here. It is generally better practice to use a URL builder where possible.